packages/components/src/errorSearchIllustration/ErrorSearchIllustration.js

```diff
@@ -0,0 +1,24 @@
+// @flow
+
+import React from 'react';
+
+const baseImagePath = './static/img/error-search-travel';
+
+export default function ErrorSearchIllustration() {
```

Comment:
It definitely shouldn't be part of this PR. But we can probably create issue after merge to combine this component with `searchIllustration` to some more universal `illustration` component. Since most code is shared between them.

Reply:
I though about the same. We will create issue for it

Reply:
Where is the issue?

Reply:
Probably nowhere when we will use images from CDN. It was about unifying illustration components.